scripts/jest/config.base.js

```diff
@@ -21,4 +21,5 @@ module.exports = {
   collectCoverageFrom: ['packages/**/*.js'],
   timers: 'fake',
   snapshotSerializers: [require.resolve('jest-snapshot-serializer-raw')],
+  testURL: 'about:blank',
```

Comment:
Note that doing this breaks in JSDOM@>=11.5 with `SecurityError: localStorage is not available for opaque origins` which is why the default has changed in Jest. Can try to do something for this when Jest 25 is released, which will force a newer version of JSDOM.

For now, I've manually made sure the entry in the lockfile is ~11.4.0, which does not have this issue

Reply:
Could you add a short TODO here?

Reply:
Done!

Reply:
ergh, expect it wasn't particularly short 😅